src/pages/workspaces/workspace/Data.js

```diff
@@ -450,9 +450,10 @@ const WorkspaceData = _.flow(
   const getRunningImportJobs = async () => {
     try {
       const runningJobs = await Ajax(signal).Workspaces.workspace(namespace, name).listImportJobs(true)
+      const currentJobIds = _.map('jobId', asyncImportJobStore.get())
       _.forEach(job => {
         const jobStatus = _.lowerCase(job.status)
-        if (!_.includes(jobStatus, ['success', 'error', 'done'])) {
+        if (!_.includes(jobStatus, ['success', 'error', 'done']) && _.indexOf(job.jobId, currentJobIds) === -1) {
```

Comment:
could also have done a `_.find` directly against `asyncImportJobStore.get()` … it felt more efficient in cases where there exist multiple jobs inside `asyncImportJobStore` to call `.get()` once and transform its results, but I'm open to feedback on this approach

Reply:
My feedback is that I like your approach. Perhaps `includes` would make this slightly clearer:
```suggestion
        if (!_.includes(jobStatus, ['success', 'error', 'done']) && !_.includes(job.jobId, currentJobIds)) {
```

Reply:
Remember that fp lodash has the arguments flipped, so `_.includes` takes haystack followed by needle; I don't think this would ever have been happy before...?

Reply:
@zarsky-broad standard lodash is haystack followed by needle (https://lodash.com/docs/4.17.15#includes), so I think this is correct as written? It works for me empirically in testing.

Reply:
ugh, my bad, brain fart. carry on...